source/appModules/eclipse.py

```diff
@@ -17,14 +22,83 @@ def event_valueChange(self):
 		# Therefore, just drop this event.
 		pass
 
+class AutocompletionListView(IAccessible):
+
+	def _get_name(self):
+		return AUTOCOMPLETION_LIST_NAME

```

Comment:
Does the autocomplete list in Eclipse not have a name itself? If it does, it'd be better to use that.

Reply:
Removed this because as you've pointed out, it is not necessary.